<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectDB</class>
 <widget class="QDialog" name="ConnectDB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>217</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>334</width>
    <height>217</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="l_ConnectionDB">
       <property name="text">
        <string>Подключение Базы Данных</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="l_NameHost">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Имя хоста</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lE_HostName">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="l_NameDB">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>16</height>
          </size>
         </property>
         <property name="text">
          <string>Имя БД</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lE_NameDB">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="l_login">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>16</height>
          </size>
         </property>
         <property name="text">
          <string>Логин</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lE_Login">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="l_Password">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>16</height>
          </size>
         </property>
         <property name="text">
          <string>Пароль</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lE_Password">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="l_DBConnectionPort">
         <property name="sizeIncrement">
          <size>
           <width>150</width>
           <height>16</height>
          </size>
         </property>
         <property name="text">
          <string>Прот подключения к БД</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sB_DBConnectionPort">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximum">
          <number>7000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConnectDB</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConnectDB</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
